app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -15,12 +15,11 @@ def initialize(http_response, config, liveness_checking_enabled = false)
           @config = config
           @http_response = http_response
           @liveness_checking_enabled = liveness_checking_enabled
-          @pii_from_doc = read_pii(true_id_product)
           super(
             success: successful_result?,
             errors: error_messages,
             extra: extra_attributes,
-            pii_from_doc: @pii_from_doc,
+            pii_from_doc: pii_from_doc,
```

Comment:
For my own understanding - I think I'm not quite sure I understand what this change gets us. Do you mind explaining the reason behind it a little more?